### PR TITLE
chore(deps): React 18, MdxEditor

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.  

**Old Announcement Editor**

image

**Updated Announcement Editor**

image

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/webapp/resources/js/components/markdown/MarkdownEditor.jsx [hunk 1]
```diff
@@ -4,91 +4,81 @@
...
+                codeMirrorPlugin({ codeBlockLanguages: { js: 'JavaScript', css: 'CSS', txt: 'text', tsx: 'TypeScript' } }),
+                linkPlugin(),
+                linkDialogPlugin(),
+                listsPlugin(),
+                frontmatterPlugin()
+            ]}
+            ref={ref}
+        />
+    );
 });
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/markdown/MarkdownEditor.jsx:84]: Fixed in 4f66162

File: src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/FastQCModal.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package ca.corefacility.bioinformatics.irida.ria.integration.components;
 
+import java.time.Duration;
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/FastQCModal.java:3]: Removed in b29a85c44a346e6fca9b0f14dc2a2e93d4102fd3

File: src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/FastQCModal.java [hunk 3]
```diff
@@ -9,6 +10,8 @@
 import org.openqa.selenium.support.PageFactory;
 
 import ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage;
+import org.openqa.selenium.support.ui.ExpectedConditions;
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/FastQCModal.java:13]: Removed in b29a85c44a346e6fca9b0f14dc2a2e93d4102fd3

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Pull request merged**